### PR TITLE
Pin iOS Podspec versions

### PR DESCRIPTION
**Pull request by kmcgill88**
Pinning the iOS versions required for Admob to work. This will address, https://github.com/YoussefKababe/admob_flutter/issues/74, for the future. If users didn't `update` their pods (which apparently Flutter doesn't do for you) you could be stuck with an old version of the ad SDK, hence it doesn't know about the new functionality.

🚀

### CODE REVIEW

File: ios/Classes/SwiftAdmobFlutterPlugin.swift [hunk 1]
```diff
@@ -36,7 +36,12 @@ public class SwiftAdmobFlutterPlugin: NSObject, FlutterPlugin {
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
     switch call.method {
     case "initialize":
-        GADMobileAds.sharedInstance().start(completionHandler: nil)
+        // https://developers.google.com/admob/ios/test-ads#enable_test_devices
+        // **iOS simulators are automatically configured as test devices.**
+        // GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers = ["You're physical iDevice Id"]
+        GADMobileAds.sharedInstance().start { (status: GADInitializationStatus) in
```
Comment (kmcgill88) [inline ios/Classes/SwiftAdmobFlutterPlugin.swift]: `GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers = [kGADSimulatorID as! String]` is not required. I've tested this out and the docs state __iOS simulators are automatically configured as test devices.__

Comment (kmcgill88) [reply ios/Classes/SwiftAdmobFlutterPlugin.swift]: Note: If you do want to add a test device. You can uncomment line 41 and put in your device id. At least until this gets implemented on the flutter side.

**Comment by YoussefKababe** [comment]
@kmcgill88 I would like to transfer this repo to you, and also set you as primary uploader on PUB, so in the future you don't need to wait for me to approve PRs and slow down contributions. Are you okay with that?

**Comment by kmcgill88** [comment]
@YoussefKababe That works for me. I'm happy to help!

**Comment by YoussefKababe** [comment]
@kmcgill88 Awesome! Seems like you already have a repository with the same name. Can you please rename/remove it so I can do the transfer?

**Comment by kmcgill88** [comment]
@YoussefKababe I just renamed my fork. It should be good to go.

**Comment by YoussefKababe** [comment]
Screen Shot 2019-10-15 at 5 22 42 AM
Any idea about this?

**Comment by kmcgill88** [comment]
@YoussefKababe  I'll try deleting my fork now. Then try again?

**Comment by YoussefKababe** [comment]
@kmcgill88 Yes let's try that!

**Comment by kmcgill88** [comment]
@YoussefKababe OK, it's gone. Give it another shot!

**Comment by YoussefKababe** [comment]
@kmcgill88 Sent a transfer request. It says it's being transferred now!

**Comment by kmcgill88** [comment]
__Moving repository to kmcgill88/admob_flutter. This may take a few minutes.__ That worked!

How does adding me to Pub work?

**Comment by YoussefKababe** [comment]
I should add you as an uploader so you can release new package versions. One minute, I'll do it now!

**Comment by YoussefKababe** [comment]
@kmcgill88 what email address do you use to access pub?

**Comment by kmcgill88** [comment]
@YoussefKababe, kevin@mcgilldevtech.com

**Comment by YoussefKababe** [comment]
@kmcgill88 Thanks! I think you should have received and invitation on your email now!

**Comment by kmcgill88** [comment]
@YoussefKababe yup, I'm in! There are a few more things I'd like to touch up before publishing `0.3.4`.

**Comment by YoussefKababe** [comment]
@kmcgill88 All right, keep the good work up! 👍

**Pull request merged**